packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -15,6 +15,10 @@ export enum ContextualMenuItemType {
   Header = 2
 }
 
+export interface IContextualMenuCallBackProps {
+  key?: string | number;
```

Comment:
We talked about creating a new interface so that you could then expand and add stuff to callbacks. I'll change it though, contextualmenuprops or state makes more sense.